src/ws/login.html

```diff
@@ -96,7 +93,11 @@ <h4 id="login-info-message" class="pf-c-alert__title"></h4>
           <a id="login-again" translate hidden>Try again</a>
         </div>
 
-        <div class="unsupported-browser" id="unsupported-browser" hidden>
+        <div class="noscript" translate>
+          Please enable JavaScript to use the Web Console.
```

Comment:
What about styling this like an error message, similar to the message about missing features?

If JSON is missing, we get this:

![image](https://user-images.githubusercontent.com/3228183/99524192-bdc14c00-29a0-11eb-89a9-4e84bd69c090.png)

When JS is disabled, it looks much plainer:

![image](https://user-images.githubusercontent.com/3228183/99524284-e5b0af80-29a0-11eb-97b1-75a38fdedd13.png)

Also, isn't it confusing to tell people to download a modern browser when they probably already have one, just with JavaScript disabled?

Reply:
> When JS is disabled, it looks much plainer

Yeah, I guess I should style that too. Good idea.

> Also, isn't it confusing to tell people to download a modern browser when they probably already have one, just with JavaScript disabled?

It depends on why they don't have JavaScript.

I thought if someone is using some weird browser or a browser that cannot handle the JS, suggesting other browsers makes sense. But perhaps it doesn't? I guess in most cases, it's due to an add-on blocking JS or sometimes even a setting in the browser to turn it off.